app/javascript/react/src/redux/configureStore.js

```diff
@@ -0,0 +1,19 @@
+import {
+  applyMiddleware,
+  configureStore,
+  getDefaultMiddleware
+} from '@reduxjs/toolkit';
+import logger from 'redux-logger';
+import messageReducer from './greeting.js';
```

Comment:
- [ ] Please ensure that the import on line 7, uses the correct name - **_greeting.js_** instead of _**message.js**_. I had to make changes to see that your app works as expected. See image below 👇🏾 

![image](https://user-images.githubusercontent.com/84156714/223201666-f62600af-d215-41ef-aee2-282a8c6f277e.png)


Well done. congratulations you did well on this project. 👏🏾

Reply:
Yes, It is, done in the last commit previous to your review. That change was not pushed.